front/pages/api/w/[wId]/assistant/agent_configurations/[aId]/index.ts

```diff
@@ -90,6 +93,44 @@ async function handler(
         });
       }
 
+      const assistantToPatch = await getAgentConfiguration(
```

Comment:
you can dry that getAgentConfiguration + some of the authorization tests for all 3 routes